daffodil-test/src/test/resources/org/apache/daffodil/section07/variables/variables.tdml

```diff
@@ -1752,7 +1751,18 @@
         </root>
       </tdml:dfdlInfoset>
     </tdml:infoset>
-  </tdml:parserTestCase>
+    <tdml:warnings>
+      <tdml:warning>Multiple choice branches</tdml:warning>
+      <tdml:warning>no required elements</tdml:warning>
+      <tdml:warning>selecting the first branch</tdml:warning>
+    </tdml:warnings>
+
+    <tdml:errors>
```

Comment:
Please add block comment to this TDML test because this is an obscure enough case that it needs some explanation. 
Suggest name of test should be "multipleBranchesWIthNoElementsDoubleWrite" or something conveying the complexity of the situation.